java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2659,12 +2661,55 @@ public T xlen(@NonNull String key) {
      *     less than the number of entries in <code>ids</code>, if the specified <code>ids</code>
      *     don't exist in the stream.
      */
-    public T xdel(String key, String[] ids) {
+    public T xdel(@NonNull String key, @NonNull String[] ids) {
```

Comment:
probably small enough to sneak in but was this intentional? changes on another command?

Reply:
Yes.  This was an intentionally sneaky move...